packages/ckeditor5-font/src/ui/colortableview.ts

```diff
@@ -293,6 +313,10 @@ export default class ColorTableView extends View {
 			this.items.add( this.documentColorsGrid );
 			this.focusTracker.add( this.documentColorsGrid.element! );
 			this._focusables.add( this.documentColorsGrid );
+
+			if ( this.colorPickerView ) {
+				this.items.add( this.colorPickerView );
```

Comment:
This logic is:

*   in `if ( this.documentColorsCount ) {` condition - this means that if the configuration `config.fontColor.documentColorsCount = 0` will hide color picker.
*   `appendGrids()` method, which is related to grids.